headers/addons/i2cdisplay.h

```diff
@@ -62,6 +62,10 @@
 #define SPLASH_DURATION 7500 // Duration in milliseconds
 #endif
 
+#ifndef DISPLAY_TURN_OFF_WHEN_SUSPENDED
```

Comment:
Nitpick, and it would be a significant refactor, but I like `DISPLAY_DISABLE_WHEN_SUSPENDED` a bit more

Reply:
That's pretty nitpicky, going to resolve this one